pkg/agent/resourcemanager/fetcher/kubelet/kubeletplugin.go

```diff
@@ -237,9 +240,19 @@ func (p *kubeletPlugin) getNumaInfo() ([]info.Node, error) {
 }
 
 func (p *kubeletPlugin) getTopologyPolicyReportContent(ctx context.Context) (*v1alpha1.ReportContent, error) {
-	topologyPolicy, err := p.topologyStatusAdapter.GetTopologyPolicy(ctx)
-	if err != nil {
-		return nil, errors.Wrap(err, "get topology policy from adapter failed")
+	var (
+		topologyPolicy nodev1alpha1.TopologyPolicy
+		err            error
+	)
+
+	if p.ormEnabled() {
```

Comment:
will this be necessary to have?

Reply:
> will this be necessary to have?

Topology policy needs to be reported in both QRM and ORM modes to ensure that the scheduler can correctly calculate node topology resources